packages/augur-ui/src/modules/market/containers/market-view.ts

```diff
@@ -1,29 +1,32 @@
-import { connect } from "react-redux";
-import { withRouter } from "react-router-dom";
-import MarketView from "modules/market/components/market-view/market-view";
-import { loadFullMarket } from "modules/markets/actions/load-full-market";
-import { selectMarket } from "modules/markets/selectors/market";
-import parseQuery from "modules/routes/helpers/parse-query";
-import { MARKET_ID_PARAM_NAME } from "modules/routes/constants/param-names";
+import { connect } from 'react-redux';
```

Comment:
```'react-redux';``` single quotes

Reply:
there are a much of single quote import stmt. we'll get that worked out when we get repo wide linting rules set.